uvicorn/main.py

```diff
@@ -175,6 +175,20 @@ def print_version(ctx: click.Context, param: click.Parameter, value: bool) -> No
     help="Enable/Disable X-Forwarded-Proto, X-Forwarded-For, X-Forwarded-Port to "
     "populate remote address info.",
 )
+@click.option(
+    "--server-header/--no-server-header",
+    "default_server_header",
```

Comment:
for consistency with other flags we have can u use `server_header` and `server_date` ?
```suggestion

```

Reply:
Resolved as suggested, thanks!